frontend/admin/src/js/components/user/UpdateUser.tsx

```diff
@@ -262,7 +262,7 @@ export const UpdateUserForm: React.FunctionComponent<UpdateUserFormProps> = ({
                   description:
                     "Placeholder displayed on the user form roles field.",
                 })}
-                options={enumToOptions(Role).map(({ value }) => ({
```

Comment:
Could you take another look at this multiselect?  I think it's no longer properly.
![image](https://user-images.githubusercontent.com/8978655/215114955-91cdd008-dd87-4671-8608-f7bfbae938de.png)

Reply:
needed to update the input name/id there, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5515/commits/879d85671237b5cbf4dee7636440f96b349f1680

Reply:
Well, it's initializing properly now but the mutation isn't working yet.  Could you take another look at this page?

Reply:
ah, whoops
mutation also needed to be re-connected as it picks fields out by name and not by formvalues as a whole
done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5515/commits/6ca00d9c547d2e47b3ef6ae391fb4a15fb88bc1b
create user I think isn't used anymore? but would've had the same issue with input naming but the mutation is fine https://github.com/GCTC-NTGC/gc-digital-talent/pull/5515/commits/ffe8b00b4a128a485060b4bbb555f96f2a65f694

Reply:
Looks good to me. :+1: